saleor/account/error_codes.py

```diff
@@ -14,6 +14,7 @@ class AccountErrorCode(Enum):
     GRAPHQL_ERROR = "graphql_error"
     INVALID = "invalid"
     INVALID_PASSWORD = "invalid_password"
+    INVALID_CREDENTIALS = "invalid_credentials"
```

Comment:
We could actually drop `INVALID_PASSWORD` at all and replace it with `INVALID_CREDENTIALS` which is more universal and if needed, it can be paired with either email or password input field.

Reply:
Do you want to do this in this PR?

Reply:
Up to you, whatever is easier/more convenient.

Reply:
Let's do this in next PR